google-ads-examples/src/main/java/com/google/ads/googleads/examples/accountmanagement/CreateAccount.java

```diff
@@ -0,0 +1,112 @@
+ *
+ * Note: this example must be run using the credentials of a Google Ads manager account. By default,
+ * the new account will only be accessible via the manager account.
+ */
+public class CreateAccount {
+
+  private static class CreateAccountParams extends CodeSampleParams {
+
+    @Parameter(
+        names = ArgumentNames.CUSTOMER_ID,
+        description = "Manager account to own the new customer.")
+    public Long managerAccountId;
+  }
+
+  public static void main(String[] args) {
+    CreateAccountParams params = new CreateAccountParams();
+    if (!params.parseArguments(args)) {
+
+      // Either pass the owning manager account on the command line or insert the ID here.
+      params.managerAccountId = Long.valueOf("INSERT_MANAGER_ID_HERE");
+    }
+
+    GoogleAdsClient googleAdsClient;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new CreateAccount().runExample(googleAdsClient, params.managerAccountId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  private void runExample(GoogleAdsClient googleAdsClient, Long managerId) {
+    // Format the current date/time to use as a timestamp in the new customer description.
+    String dateTime = ZonedDateTime.now().format(DateTimeFormatter.RFC_1123_DATE_TIME);
+
+    // Initialize a Customer object to be created.
+    Customer customer = Customer.newBuilder()
+        .setDescriptiveName(
+            StringValue.of("Account created with CustomerService on '" + dateTime + "'"))
+        .setCurrencyCode(StringValue.of("USD"))
+        .setTimeZone(StringValue.of("America/New_York"))
+        .setTrackingUrlTemplate(StringValue.of("{lpurl}?device={device}"))
+        .setFinalUrlSuffix(
+            StringValue.of("keyword={keyword}&matchtype={matchtype}&adgroupid={adgroupid}"))
+        .setHasPartnersBadge(BoolValue.of(false))
+        .build();
+
+    // Connect the CustomerService and send the createCustomer request.
```

Comment:
"Connect the CustomerService" sounds a bit odd. How about just "Sends the request to create the customer."

Reply:
Done